transports/msmq/index.md

```diff
@@ -156,6 +156,12 @@ See also [Message Queuing Security Overview](https://docs.microsoft.com/en-us/pr
 
 ## Distributed Transaction Coordinator
 
-To support guaranteed [once delivery of messages,](/nservicebus/operations/transactions-message-processing.md) NServiceBus makes use of the Distributed Transaction Coordinator (DTC) to synchronize transactions between MSMQ and the database. For this to work, the DTC must be started and configured correctly.
+NServiceBus makes use of the Microsoft Distributed Transaction Coordinator (MSDTC) to synchronize transactions between MSMQ and the database in order to support [guaranteed once delivery of messages](/nservicebus/operations/transactions-message-processing.md). For this to work, the MSDTC must be started and configured correctly. This can be done manually or with the [NServiceBus PowerShell module](/nservicebus/operations/management-using-powershell.md).
```

Comment:
```suggestion
NServiceBus makes use of the Microsoft Distributed Transaction Coordinator (MSDTC) to synchronize transactions between MSMQ and the database in order to support [guaranteed once delivery of messages](/nservicebus/operations/transactions-message-processing.md). For this to work, MSDTC must be started and configured correctly. This can be done manually or using the [NServiceBus PowerShell module](/nservicebus/operations/management-using-powershell.md).
```